Dockerfile

```diff
@@ -9,21 +9,24 @@ RUN apt-get update && apt-get install -y \
 
 ARG CEPH_REPO_URL=https://download.ceph.com/debian-nautilus/
 RUN wget -q -O- 'https://download.ceph.com/keys/release.asc' | apt-key add -
-RUN apt-add-repository "deb ${CEPH_REPO_URL} xenial main"
+RUN true && \
+  apt-add-repository "deb ${CEPH_REPO_URL} xenial main" && \
+  apt-get update && \
+  apt-get install -y ceph libcephfs-dev librados-dev librbd-dev curl gcc g++
 
-RUN add-apt-repository -y ppa:gophers/archive
-
-RUN apt-get update && apt-get install -y \
-  ceph \
-  libcephfs-dev \
-  librados-dev \
-  librbd-dev \
-  golang-1.10-go
+ENV GOTAR=go1.12.16.linux-amd64.tar.gz
```

Comment:
Instead of an `ENV`, `ARG` might be better?
- https://docs.docker.com/engine/reference/builder/#arg

This looks good to me though, and when we make selecting a Go version more dynamic, we'll probably need `ARG`.